### ISSUE TITLE
User would like to be able to filter also ranges of kW to get a better overview 

### ISSUE THREAD

**Issue by ohnewein** [noi-techpark/webcomp-mobility-echarging-map#60]
User request: der Filter der Karte https://webcomponents.opendatahub.bz.it/webcomponent/98013c75-f27c-414e-aff9-c58ed760e791 erlaubt es nur genau nach einem bestimmten kW Wert zu suchen. Also z.B: alle 100kW Stationen . Aber nicht "alle größer gleich 100 kW"

**Comment by dulvui** [comment]
Problem fixed and running on testing
https://webcomponents.opendatahub.testingmachine.eu/webcomponent/98013c75-f27c-414e-aff9-c58ed760e791

**Comment by ohnewein** [comment]
Was it about "max power" or rather "min power". As far as I understand the request, the user was looking for stations delivering at least X kW.

**Comment by gufmar** [comment]
The technical filter functionality seems to fit. What is still irritating is the designation "max power".
It would now be "min power" in the sense of "search results with at least the specified value".

**Comment by dulvui** [comment]
Yes you are right, min power fits better. I think it was called max power because we have the data field called maxPower.
Now it is online also in production.
